scania-digital-design-system/tegel
# fix(dropdown): adding check for value

**Describe pull-request**  
Fixing errors in console when choosing in dropdown when filter is not set. Also included fix for problem with clicking chevron and opening dropdown. 

**Solving issue**  
Fixes: [CDEP-3098](https://tegel.atlassian.net/jira/software/projects/CDEP/boards/1?selectedIssue=CDEP-3098)

**How to test**  
1. Go to Storybook
2. Check in dropdown
3. Select a value from dropdown, make sure that no error is logged in web inspector console.



[CDEP-3098]: https://tegel.atlassian.net/browse/CDEP-3098?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
